dependent/agriculture-and-food-security.csl

```diff
@@ -1,18 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
-  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/bmc -->
+  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer -->
   <info>
     <title>Agriculture &amp; Food Security</title>
-    <title-short>Agric Food Secur</title-short>
     <id>http://www.zotero.org/styles/agriculture-and-food-security</id>
     <link href="http://www.zotero.org/styles/agriculture-and-food-security" rel="self"/>
-    <link href="http://www.zotero.org/styles/biomed-central" rel="independent-parent"/>
-    <link href="http://www.agricultureandfoodsecurity.com" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/springer-basic-brackets" rel="independent-parent"/>
```

Comment:
Yeah, it looks like most of the styles should be connected to the `springer-vancouver-brackets.csl` and not `springer-basic-number.csl`. How can we deal with that?

Moreover, the shorttitle and field(s) are not present anymore. Shouldn't we try to add them again, maybe using a similar approach in Excel as before?